database/dynamodb/util.go

```diff
@@ -363,13 +364,13 @@ func (db *DynamoDB) deleteTask(ctx context.Context, id string) error {
 						"index": res["index"],
 					},
 				}
-				// TODO handle error
-				db.client.DeleteItem(item)
-			}
-			if page.LastEvaluatedKey == nil {
-				return false
+				// TODO handle error without panic
+				_, err := db.client.DeleteItem(item)
+				if err != nil {
+					log.Fatalf("failed to delete content item: %v", err)
```

Comment:
Better than a panic:

```suggestion
					log.Printf("ERROR: failed to delete content item: %v\n", err)
```